<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>CM:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="second_spinbox">
     <property name="maximum">
      <number>6000</number>
     </property>
     <property name="value">
      <number>600</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="suffix_line_edit"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cm_combobox"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>PV Suffix:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Seconds:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="PyDMArchiverTimePlot" name="plot">
     <property name="toolTip">
      <string/>
     </property>
     <property name="updateMode" stdset="0">
      <enum>PyDMTimePlot::AtFixedRate</enum>
     </property>
     <property name="timeSpan">
      <double>600.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMTimePlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.timeplot</header>
  </customwidget>
  <customwidget>
   <class>PyDMArchiverTimePlot</class>
   <extends>PyDMTimePlot</extends>
   <header>pydm.widgets.archiver_time_plot</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
